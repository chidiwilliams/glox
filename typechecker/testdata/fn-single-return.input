var fn = fun (firstName: string, lastName: string): number {
	return firstName + lastName;
};
fn;
