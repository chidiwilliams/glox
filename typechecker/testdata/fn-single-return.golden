type error on line 1: expected enclosing function to return 'number', but got 'string'
